app/models.py

```diff
@@ -265,3 +264,25 @@ class InvitedUser(db.Model):
         default=datetime.datetime.now)
     status = db.Column(
         db.Enum(*INVITED_USER_STATUS_TYPES, name='invited_users_status_types'), nullable=False, default='pending')
+
+
+class Permission(db.Model):
+    __tablename__ = 'permissions'
```

Comment:
Probably need to model permissions as a list - like we do with user status etc - enums?